neurom/stats.py

```diff
@@ -176,3 +176,26 @@ def compare_two(data1, data2, test=StatTests.ks):
     Stats = namedtuple('Stats', ['dist', 'pvalue'])
 
     return Stats(*results)
+
+
+def total_score(paired_dats, p=2, test=StatTests.ks):
```

Comment:
Does it make sense for `p` to be an input parameter? Can we get different, meaningful results by changing only p?